api/main.py

```diff
@@ -1,31 +1,39 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 #
-# Copyright (C) 2021 Collabora Limited
+# Copyright (C) 2021, 2022 Collabora Limited
 # Author: Guillaume Tucker <guillaume.tucker@collabora.com>
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
 
+# pylint: disable=unused-argument disable=global-statement
+
+"""KernelCI API main module"""
+
+from typing import List
 from fastapi import Depends, FastAPI, HTTPException, status, Request
 from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from bson import ObjectId, errors
 from .auth import Authentication, Token
 from .db import Database
 from .models import Node, User, Password
 from .pubsub import PubSub, Subscription
-from typing import List
-from bson import ObjectId, errors
 
 app = FastAPI()
 db = Database()
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
 auth = Authentication(db)
-pubsub = None
+PUBSUB = None
```

Comment:
Why do we have this name in capitals?
PEP8 suggests that capitals should be used when we're declaring a  constant.

Reply:
> Why do we have this name in capitals? PEP8 suggests that capitals should be used when we're declaring a constant.

Pylint was complaining about it with "invalid-name" error.

Reply:
Can you link the pylint logs?

Reply:
Here are the logs:
```
api/main.py:19:0: C0103: Constant name "pubsub" doesn't conform to UPPER_CASE naming style (invalid-name)
api/main.py:24:4: C0103: Constant name "pubsub" doesn't conform to UPPER_CASE naming style (invalid-name)
```

Reply:
It's definitely a variable, hence it shouldn't be all capitals. I think it's reasonable to add `pubsub` to `good-names` in pylint configuration or simply turn off the warning for this particular line with a `# pylint: disable=C0103` comment

Reply:
Done.